readthedocs/search/tasks.py

```diff
@@ -212,3 +212,27 @@ def record_search_query(project_slug, version_slug, query, total_results, time_s
     )
     obj.created = time
     obj.save()
+
+
+@app.task(queue='web')
+def increase_page_view_count(project_slug, version_slug, path):
+    today_date = timezone.now().date()
+    page_view_obj = PageView.objects.filter(
+        project__slug=project_slug,
+        version__slug=version_slug,
+        path=path,
+        date=today_date,
+    ).first()
+
+    if page_view_obj:
+        page_view_obj.view_count += 1
```

Comment:
This should use an [F expression](https://docs.djangoproject.com/en/1.11/ref/models/expressions/#f-expressions) to avoid a race condition.